assets/training/finetune_acft_hf_nlp/environments/acpt-grpo/context/Dockerfile

```diff
@@ -0,0 +1,19 @@
+#PTCA image
+FROM mcr.microsoft.com/aifx/acpt/stable-ubuntu2204-cu126-py310-torch271:{{latest-image-tag:biweekly\.\d{6}\.\d{1}.*}}
+
+USER root
+
+RUN apt-get update && apt-get -y upgrade
+RUN pip install --upgrade pip
+
+COPY requirements.txt .
+RUN pip install -r requirements.txt --no-cache-dir
+
+RUN pip install azureml-acft-common-components=={{latest-pypi-version}}
+RUN pip install azureml-sdk==1.60.0
```

Comment:
absolutely no for azureml-sdk. That is a metapackage for lazy data scientists. Pick the actual package you need. If you need azureml-core, please explain why can't you use v2 sdk instead.